### ISSUE TITLE
comma-dangle for dynamic import

### ISSUE THREAD

**Issue by alfredking1988** [eslint/eslint#11310]
**Environment**

* **ESLint Version: 5.12.1**
* **Node Version: 10.15.0** 
* **npm Version: 6.6.0**

**Parser: `babel-eslint`**

**Configuration:**

```js
'comma-dangle': ['warn', {
  arrays: 'always-multiline',
  objects: 'always-multiline',
  imports: 'always-multiline',
  exports: 'always-multiline',
  functions: 'always-multiline',
}],
```

**Source code causing the issue.**

```js
() => import(
  /* webpackChunkName: "ImageCropper" */
  './ImageCropper.vue'  // no trailing comma here
)
```

**What did you expect to happen?**
I want the trailing comma style. But as the babel/babel/issues/8957 already said: 

> **Dynamic** imports must **disallow** trailing comma

So I want the above dynamic import code, which without trailing comma, pass the lint.

**What actually happened?**
The eslint says `Missing trailing comma`.

**Comment by kaicataldo** [comment]
Thanks for the issue. As dynamic imports are Stage 3, I do think we should make sure the `comma-dangle` rule does the right thing here. That being said, it's not clear to me what the correct behavior actually is. While Babel's parser now errors on this, there doesn't seems to actually be consensus in [this discussion](https://github.com/tc39/proposal-dynamic-import/issues/15) (which is referenced in the Babel issue above). 

Maybe @ljharb could clarify this for us?

**Comment by ljharb** [comment]
I believe this is correct and that the syntax does not allow it, so the eslint rule should not be warning there (just like after rest args).

See https://github.com/tc39/proposal-dynamic-import/issues/15

**Comment by kaicataldo** [comment]
I'm realizing that this might have to actually be a change in [eslint-plugin-babel](https://github.com/babel/eslint-plugin-babel), as our policy is that [rules will not crash on stage 3 features](https://github.com/eslint/eslint#what-about-experimental-features), and that we only update rules for new syntax once they become stage 4.

This shouldn't be a too difficult to do with https://github.com/not-an-aardvark/eslint-rule-composer.
